homeassistant/components/cover/rflink.py

```diff
@@ -0,0 +1,93 @@
+"""
+Support for Rflink switches.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/switch.rflink/
+"""
+import asyncio
+import logging
+
+
+from homeassistant.components.rflink import (
+    DATA_ENTITY_GROUP_LOOKUP, DATA_ENTITY_LOOKUP,
+    DEVICE_DEFAULTS_SCHEMA, DOMAIN, EVENT_KEY_COMMAND, CoverableRflinkDevice,
+    cv, vol)
```

Comment:
This seems correct for the RFlink hub, though al other sub components like, sensor, switch, light uses it this way.
If you are sure this is the correct way i'll change my code.